api/envoy/config/transport_socket/capture/v2/capture.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package envoy.config.transport_socket.capture.v2;
```

Comment:
Can we put this into v2alpha? I think with the tap stuff I'm going to start working on there is likely to be some evolution as we figure out all of this.